Core.UnitTests/ReadInput/InputReaderTests.cs

```diff
@@ -28,6 +28,50 @@ namespace Remotion.ReleaseProcessAutomation.UnitTests.ReadInput;
 [TestFixture]
 internal class InputReaderTests
 {
+  [Test]
+  [TestCase("y", true)]
+  [TestCase("Y", true)]
+  [TestCase("n", false)]
+  [TestCase("N", false)]
+  public void ReadConfirmation_WithInput_ReturnsOutput (string inputText, bool output)
+  {
+    var testConsole = new TestConsole();
+    testConsole.Input.PushTextWithEnter(inputText);
+    var inputReader = new InputReader(testConsole);
+
+    var act = inputReader.ReadConfirmation();
+
+    Assert.That(act, Is.EqualTo(output));
+  }
+
+  [Test]
+  public void ReadConfirmation_WithImproperInput_RepeatsInput ()
+  {
+    var testConsole = new TestConsole();
+    testConsole.Input.PushTextWithEnter("notACorrectInput");
+    testConsole.Input.PushTextWithEnter("y");
+    var inputReader = new InputReader(testConsole);
+
+    var act = inputReader.ReadConfirmation();
+
+    Assert.That(testConsole.Output, Does.Contain("The input 'notACorrectInput' is not a valid option."));
+    Assert.That(act, Is.EqualTo(true));
+  }
+
+  [Test]
+  [TestCase(true)]
+  [TestCase(false)]
+  public void ReadConfirmation_WithEnterInput_ReturnsDefault (bool defaultVal)
```

Comment:
Naming things: please do not use abreviations. "value", not "val"; "actual" not "act".

Reply:
"act" was actually meant as the noun act and not an abbreviation, but it was a bad name regardless.